gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/artifacts/DependencyDiscovery.java

```diff
@@ -43,16 +40,17 @@ public static ArtifactView getFilteredArtifact(Project project, Configuration co
     }
 
     public static Configuration copyConfiguration(Project project, String configurationName, String name) {
-        Configuration conf = project.getConfigurations()
-                .create(GUtil.toLowerCamelCase(configurationName + " for " + name), c -> {
-                    c.setCanBeConsumed(true);
-                    c.setCanBeResolved(true);
-                    c.setVisible(false);
+        Configuration consumable = project.getConfigurations()
+                .create(GUtil.toLowerCamelCase(configurationName + " for " + name), conf -> {
```

Comment:
hmmm, didn't realise that we can make configurations with spaces.

Reply:
Unless `GUtil.toLowerCamelCase` strips the spaces?

Reply:
it converts the sentence to lowerCamelCase so the spaces go away :)